### PR TITLE
add description for teamsapp permission

### PR DESCRIPTION
**Pull request by bhartono**

### CODE REVIEW

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -1480,6 +1480,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 | _TeamsActivity.Read.All_ | Read all users' teamwork activity feed | Allows the app to read all users' teamwork activity feed, without a signed-in user. This permission is currently in private preview only and is not available for public use.	| Yes | No |
 | _TeamsActivity.Send_ | Send a teamwork activity to any user | Allows the app to send new activities to any users' teamwork activity feed, without a signed-in user. This permission is currently in private preview only and is not available for public use.	| Yes | No |
 
+## Teams App permissions
+
+#### Delegated permissions
+
+|   Permission    |  Display String   |  Description | Admin Consent Required | Microsoft Account supported |
+|:----------------|:------------------|:-------------|:-----------------------|:--------------|
+| _TeamsApp.Read.All_ | Allows the app to read the Teams apps that are installed for the signed-in user, and in all teams the user is a member of. Does not give the ability to read application-specific settings. This permission is currently in private preview only and is not available for public use.	| No | No |
```
Comment (nkramer) [inline concepts/permissions-reference.md]: This permission requires admin consent.

File: concepts/permissions-reference.md [hunk 2]
```diff
@@ -1480,6 +1480,22 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+| _TeamsApp.ReadWrite.All_ | Allows the app to read, install, upgrade, and uninstall Teams apps, on behalf of the signed-in user and also for teams the user is a member of. Does not give the ability to read or write application-specific settings. This permission is currently in private preview only and is not available for public use.	| No | No |
```
Comment (nkramer) [inline concepts/permissions-reference.md]: This permission requires admin consent.

Comment (bhartono) [reply concepts/permissions-reference.md]: fixed

File: concepts/permissions-reference.md (hunk 1 above)
Comment (bhartono) [reply concepts/permissions-reference.md]: fixed

**Pull request merged**